common/src/main/java/org/sonarlint/intellij/common/util/SonarLintUtils.java

```diff
@@ -47,8 +48,9 @@
 public class SonarLintUtils {
 
   private static final Logger LOG = Logger.getInstance(SonarLintUtils.class);
-  private static final String[] SONARCLOUD_ALIAS = {"https://sonarqube.com", "https://www.sonarqube.com",
-    "https://www.sonarcloud.io", "https://sonarcloud.io"};
+  public static final String SONARCLOUD_URL = System.getProperty("sonarlint.internal.sonarcloud.url", "https://sonarcloud.io");
+  private static final Set<String> SONARCLOUD_ALIAS = Set.copyOf(List.of("https://sonarqube.com", "https://www.sonarqube.com",
```

Comment:
Why not directly create a Set with `Set.of`?

Reply:
`Set.of` throws when there are duplicates in the arguments. What I want to do here is to keep `https://sonarcloud.io` in all cases, plus the custom one that might be configured